package-lock.json

```diff
@@ -1,7 +1,7 @@
 {
 	"name": "@preact/preset-vite",
 	"version": "2.7.0",
-	"lockfileVersion": 2,
+	"lockfileVersion": 3,
```

Comment:
This seems to change the version of the lockfile, not sure if there are any bad changes in here, doesn't look like it, just wanted to flag. Maybe it's worth switching this repo to `pnpm` later as well

Reply:
Yeah 2 -> 3 is a product of NPM 9+. Backwards compatible to v7 if need be.

Didn't have an older version of NPM available but will try to figure that out. Needless bump otherwise.

Reply:
Reverted